modules/exploits/linux/http/pulse_secure_file_disclosure.rb

```diff
@@ -0,0 +1,73 @@
+# Quick hackish exploit for the CVE-2019-11510
+# made with help from Alyssa Herrera
+# Sorry for using your exploit as a template Carlos E. Vieira
+# Grabs /etc/passwd as a proof of concept
+
+require 'msf/core'
+class MetasploitModule < Msf::Auxiliary
+	include Msf::Exploit::Remote::HttpClient
+	include Msf::Post::File
+	def initialize(info = {})
+		super(update_info(info,
+			'Name'           => 'Pulse Secure - System file leak',
```

Comment:
Traditionally, the module name describes the vulnerable software, vulnerable functionality, and vulnerability type.

Suggest something like:

```suggestion
			'Name'           => 'Pulse Secure SSL VPN guacamole Path Traversal',
```